lua/neogit/lib/util.lua

```diff
@@ -124,7 +124,7 @@ return {
   filter = filter,
   str_right_pad = str_right_pad,
   str_count = str_count,
-  create_fold = create_fold,
+  --create_fold = create_fold, -- create_fold is not defined
```

Comment:
I couldn't find the definition of that function

Reply:
Ah yeah you can remove this.